custom/icds/tasks/data_pulls.py

```diff
@@ -34,10 +37,23 @@ def run_data_pull(data_pull_slug, month, location_id=None, email=None):
         raise
     else:
         if email and filename:
+            db = get_blob_db()
+            download_id = DownloadBase.new_id_prefix + make_uuid()
+            with open(filename, 'rb') as _file:
+                db.put(
+                    _file,
+                    domain=domain,
+                    parent_id=domain,
+                    type_code=CODES.data_export,
+                    key=download_id,
+                    timeout=24 * 60,
+                )
             exposed_download = expose_blob_download(
-                filename, expiry=24 * 60 * 60,
+                filename, expiry=24 * 60,
```

Comment:
Do you want this to expire in 24 minutes? Looks like the default is 24 hours:

https://github.com/dimagi/commcare-hq/blob/bda8b8d1247271cf17d076c33749cca68e674c2e/corehq/ex-submodules/soil/__init__.py#L96

Unfortunately that default cannot be used here since there are multiple functions between here and there and some of the arguments do not have the default.

Reply:
oops thanks, going to keep this `24 * 60 * 60` so that just in case we change the default, this stays the same.